worker/upgradeseries/worker_test.go

```diff
@@ -147,34 +121,6 @@ func (s *workerSuite) expectMachineUnitServiceDiscovery() {
 	s.expectServiceDiscovery(false)
 }
 
-func (s *workerSuite) TestMachinePrepareMachineUnitFilesWrittenProgressPrepareComplete(c *gc.C) {
```

Comment:
This shouldn't be deleted. We need to detect that when the machine is "prepare_started" and all units are "prepare-completed", that we invoke the upgrader and progress the machine to "prepare-completed". 

So the names would become `TestMachinePrepareStartedUnitFilesWrittenProgressPrepareComplete` and `expectMachinePrepareStartedUnitFilesWrittenProgressPrepareComplete` and we just change the first expectation to `exp.MachineStatus().Return(model.UpgradeSeriesPrepareStarted, nil)`